exporter/stats/stackdriver/stackdriver.go

```diff
@@ -328,18 +332,18 @@ func namespacedViewName(v string, escaped bool) string {
 func newLabels(tags []tag.Tag) map[string]string {
 	labels := make(map[string]string)
 	for _, tag := range tags {
-		labels[tag.Key.Name()] = tag.Value
+		labels[sanitize(tag.Key.Name())] = tag.Value
 	}
 	return labels
 }
 
 func newLabelDescriptors(keys []tag.Key) []*labelpb.LabelDescriptor {
-	labelDescriptors := make([]*labelpb.LabelDescriptor, len(keys))
+	labelDescriptors := make([]*labelpb.LabelDescriptor, 0, len(keys))
```

Comment:
Or you can just do

for i, key := range keys {
labelDescriptors[i] = ...

to avoid append all together and not caring about size vs capacity when making the slice.